assets/js/components/settings/settings-module.js

```diff
@@ -363,8 +366,8 @@ class SettingsModule extends Component {
 											{ isEditing[ moduleKey ] || isSavingModule ? (
 												<Fragment>
 													<Button
-														onClick={ () => handleEdit( moduleKey, hasSettings && setupComplete ? 'confirm' : 'cancel' ) }
-														disabled={ isSavingModule }
+														onClick={ () => handleEdit( moduleKey, hasSettings && setupComplete ? 'confirm' : 'cancel', hasSettings && setupComplete ? submitChanges : null ) }
```

Comment:
The multiple ternaries on the same line as arguments is quite tough to read here. I know it wasn't in the IB but I think this set of params would benefit from being defined above, before the JSX render as variables that can be passed to `handleEdit`. As-is, this is too hard to read—even before it was crowded, but I think the extra argument makes it worth it to define these params above and use a more concise call to `handleEdit` here.